weed/s3api/s3api_account.go

```diff
@@ -0,0 +1,69 @@
+package s3api
```

Comment:
Is it recommended to create a separate directory `s3_account` for `s3_account.go`, because if the shell command refers to `AccountAnonymous` or `AccountAdmin`, there may be a circular reference (s3api->command->s3api, now s3api has a reference to the command directory ); and if the management function of Account is added, it can also be placed in this directory